### ISSUE TITLE
Hobelmaschine hat erste Scharten

### ISSUE THREAD

**Issue by LukasFreeze** [comakingspace/do-something#243]
Gestern ist bei der Einweisung zur Abricht- und Dickenhobelmaschine aufgefallen, dass die Messer an einer Stelle schon eine ordentliche Scharte haben. Da ist wohl jemand in einen Nagel gefahren?! :-(

Der Vorschub wird nicht merklich erschwert und man kann trotz diesem Schaden gut hobeln, aber optisch ist das Ergebnis natürlich nicht ideal:
![IMG_20210909_135947_083](https://user-images.githubusercontent.com/19755883/132682589-c427bc2f-e4fc-4308-a245-c5edbb6982e8.jpg)

Ein paar kleinere Scharten sind über die ganze Breite verteilt, evtl sieht man das auf diesem Foto durch die mobile Bearbeitung aber nicht mehr.

Was ist zu tun: erstmal lohnt es sich wohl nicht, gleich alles zu demontieren und einen Schärfversuch zu starten, da sicher noch weitere Scharten dazukommen (und "dürfen", bevor die Maschine so keinen Spaß mehr macht). @produkt-manager will sich aber schon einmal schlau machen, damit wir es irgendwann versuchen könnten.

Workarounds für Teile, die *richtig* plan sein sollen:
- zuletzt nochmal vorsichtig von Hand drüberhobeln
- schmale Stücke: Anschlag ca. 5 cm vorziehen und auf den vorderen/"linken" 2/3 des Messers abrichten (Dickenhobel dementsprechend eher "rechts" nutzen)

**Comment by produkt-manager** [comment]
Die genauen Maschinendaten müßte man genau checken, und auch nochmal genauere Preise recherchieren - prinzipielle Angaben:

- Laut Handbuch sind doppelseitige Hobeleisen verbaut (Kapitel 11.5) https://www.felder-group.com/WEB/Machines/Manuals/A3-26_e-classic/504010-901_01_1706_HA_BA_A3-26_A3-31_A3-41_GER_Lo.pdf, d.h man kann sie wenden. 
- Um die Hobeleisen aus-/ einzuzubauen benötigt man eine Lehre, die man entweder bauen kann, wie im Handbuch beschrieben, oder kaufen kann - Beispiel Magnetversion ca 100.-  https://www.felder-group.com/de-at/shop/einstellgeraet-sc617959/streifenhobelmesser-magnet-einstellgerat-sp617958
- Die Hobeleisen verschleißen (Werden stumpf), oder bekommen Scharten, wie unsere. Man kann sie schleifen, wobei der Preis sich nach den Schäden richtet. Pro Satz kostet das mind 60.- https://www.pwwu24.de/hobelmesser-schaerfen?number=S10173.4, und man muss die Messer für eine Zeit einschicken.
- Neukauf 150.- beim Hersteller (Einheit 6 Stück https://www.felder-group.com/de-at/shop/systemhobelmesser-sc124630/hs-m42-systemhobelmesser-sp124640), bzw billiger für Nachbauten: https://www.mima.de/Marken/F/FELDER/Felder-Systemhobelmesser-in-HS-M42-SS-Stahlqualitaet.html
- Selbstschleifen ist möglich und auch nicht weiter kompliziert, wenn man eine Lehre hat oder baut, die das Blatt in einem Winkel hält, und wenn man einen Abziehstein aus Leder hat + einen Schleifstein für HSS Stahl. Angebot inclusive Anwendungsvideo https://www.tormek.com/austria/de/schleifvorrichtungen/svh-320-vorrichtung-fuer-maschinenhobelmesser/. Fertige Lehre ca 150.- . DIY Idee hier (Metallbau) https://forum.zerspanungsbude.net/viewtopic.php?t=24088 
- Es gibt einen anderen Walzentyp der mit Plättchen arbeitet, die man einzel tauschen kann https://www.felder-group.com/de-at/shop/silent-power-hobelmesser-sc124679/hw-ersatzmesser-fur-silent-power-spiralmesser-hobelwelle-sp124683

Fazit
- Eine Einbaulehre benötigen wir in jedem Fall (bauen oder kaufen)
- Erst einmal können wir unsere Hobeleisen noch ein mal umdrehen (vermutlich)
- Schleiflehre für 150.- + Selbstschleifen dürfte langfristig die preiswerteste Methode sein, wenn es einen passenden Schleifstein bereits gibt. 
- Auftragsschleifen als Notlösung, wenn man die Maschine eine Zeit nicht benötigt

**Comment by LarsKumbier** [comment]
Ja, der Transport hat auch teilweise funktioniert - aber alles sehr 
stockend. Anpressdruck habe ich auch variiert (durch weniger oder mehr 
Abtrag pro Durchgang) ohne Erfolg.

**Comment by LukasFreeze** [comment]
Ich habe letzte Woche eine Bohle abgerichtet und keine Veerschlechterung seit meinem ursprünglichen Post feststellen können. Mit Ausnahme der bekannten Scharte am rechten Rand sah die Oberfläche super aus und der Gegendruck von der Maschine war meinem Gefühl nach so stark wie bisher.
Den automatischen Vorschub im Dickenhobel habe ich dieses Mal nicht getestet, aber wenn es da neue Probleme gibt würde ich nicht auf die Messer als Ursache tippen

**Comment by Pakue95** [comment]
Update: Die Messer wurden vor gut einem Monat gewechselt, leider sind schon wieder Scharten drin :(

**Comment by LukasFreeze** [comment]
@Pakue95 wir hatten langfristig ja auch mal über eine Helix-Messerwelle nachgedacht, aber du hattest an anderer Stelle gesagt dass ein Umbau doch nicht ohne weiteres möglich ist und am besten (oder zwingend?) eine neue Maschine hermüsste.
Sind die Kosten trotzdem irgendwie auf 100 € genau bezifferbar, damit wir jetzt schonmal einen Bedarf auf betterplace anlegen könnten? Der könnte sich dann schonmal langsam füllen, wann immer sich jemand über eine Scharte ärgert und/oder für eine verantwortlich fühlt...

**Comment by Pakue95** [comment]
Nein, die Kosten kann ich nicht beziffern, sind aber laut Felder so teuer, dass es preiswerter wäre diese zu verkaufen und eine neue mit Silent Spin zu bestellen.

**Comment by LukasFreeze** [comment]
Schade, dann hatte ich mir das nicht eingebildet 😜
Meinst du denn wir sollten so einen großen Austausch generell anpeilen oder ist das die Mühe verglichen mit häufigen Messerwechseln noch nicht wert? Falls wir es im Prinzip gut finden kann man über die Finanzierung ja nochmal nachdenken - ich meine auch auf keinen Fall dass hier im nächsten halben Jahr was passieren müsste, aber wenn die Perspektive da ist finden sich ja hoffentlich schon vorab Unterstützer 🙂

**Comment by Pakue95** [comment]
Häufige Messerwechsel? Bisher hat noch niemand nen Cent für neue Messer gegeben, daher würde ich mal davon ausgehen dass die Scharten auch keinen so sehr stören. Würde da momentan nichts tun.

Die Scharten sind Nutzerfehler weil *reclaimed wood* darauf wieder aufbereitet wurde ohne sicherzustellen, dass keine Nägel oÄ drin sind. Das helfen uns Karbidklingen auch nur oberflächlich.

**Comment by LukasFreeze** [comment]
Alles klar, darauf wollte ich im Wesentlichen hinaus: *wenn* dann mal jemand Geld für bessere Oberflächen investieren möchte wäre die Option also bis auf weiteres, neue Messer zu kaufen.

**Comment by NitramLegov** [comment]
Wenn wir betterplace Bedarfe anlegen wollen, die signifikante Werte haben und von Anfang an langfristig ausgelegt sind sollten wir vorher auch nochmal kurz über die administrative Abwicklung nachdenken.

Bei Betterplace müssen wir das Geld spätestens alle 12 Monate abrufen: https://www.betterplace.org/c/hilfe/zeit-projekt-zu-finanzieren

Ich sehe nicht, dass ein solches Thema in 12 Monaten finanziert ist. Das heißt wir müssen das Geld entsprechend bei uns zurücklegen. Da sollten wir nochmal drüber sprechen, wie wir das machen und vom Rest trennen.
